cloudify_gcp/compute/network.py

```diff
@@ -13,8 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from googleapiclient.errors import HttpError
+
 from cloudify import ctx
```

Comment:
@GhalebKhaled can you move this ``from googleapiclient.errors import HttpError`` import to the cloudify import sections since these are all third parties import